include/powsybl/iidm/StaticVarCompensator.hpp

```diff
@@ -67,6 +78,8 @@ class StaticVarCompensator : public Injection {
 
     std::vector<double> m_reactivePowerSetpoint;
 
+    std::reference_wrapper<Terminal> m_regulatingTerminal;
```

Comment:
The Generator implementation is not compliant with this one, but this one seems better. Could you create another pull request to fix the generator implementation?